tests/unit/test_install.py

```diff
@@ -0,0 +1,155 @@
+import webbrowser
+from unittest.mock import MagicMock, create_autospec
+
+import pytest
+from databricks.labs.blueprint.installation import Installation, MockInstallation
+from databricks.labs.blueprint.tui import MockPrompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.install import WorkspaceInstallation, WorkspaceInstaller
+
+
+@pytest.fixture
+def ws():
+    ws = create_autospec(WorkspaceClient)
+
+    ws.product = "remorph"
+    ws.product_version = __version__
+
+    return ws
+
+
+@pytest.fixture
+def mock_installation():
+    return MockInstallation(
+        {"state.json": {"version": 1, "source": "snowflake", "skip_validation": True, "catalog_name": "remorph_test"}}
+    )
+
+
+def test_install(ws, mock_installation):
+    prompts = MockPrompts(
+        {
+            r"Select the source": "0",
+            r"Do you want to Skip Validation": "yes",
+            r"Enter catalog_name": "remorph_test",
+            r"Enter schema_name": "remorph_schema",
+            r".*": "",
+        }
+    )
+    install = WorkspaceInstaller(prompts, mock_installation, ws)
+    install.run()
```

Comment:
Where are assertions that verify methods called/not called?

Reply:
Added asserts